packages/osd-test-subj-selector/index.ts

```diff
@@ -28,15 +28,15 @@
  * under the License.
  */
 
-function selectorToTerms(selector) {
+function selectorToTerms(selector: string) {
```

Comment:
Yeah,I don't think this module is getting correctly parsed or built as typescript. For now, I'll just change it back to `.js`, because I don't know enough about the building and bundling yet to fix it quickly.

Reply:
fixed